src/auth/auth.service.ts

```diff
@@ -29,21 +33,38 @@ export class AuthService {
     return this.jwtService.verifyAsync(jwtToken);
   }
 
-  async signIn(signInRequest: SignInRequest): Promise<string> {
-    const user = await this.userRepository.getUserByEmailWithPassword(
-      signInRequest.email,
+  async generateBearerToken(id: number, email: string): Promise<string> {
+    return this.jwtService.signAsync(
+      {
+        id,
+        email,
+      },
+      {
+        secret: process.env.JWT_SECRET,
+        expiresIn: `${process.env.JWT_EXPIRY_TIME}s`,
```

Comment:
I wouldn't hardcode a limit to seconds. I would set values in envs like `60s` or `2h` or `7d` etc.